server/models/LegalDocument.ts

```diff
@@ -144,6 +145,61 @@ class LegalDocument extends Model<InferAttributes<LegalDocument>, InferCreationA
     });
   };
 
+  /**
+   * To use when a host admin manually provides an external link to a tax form.
+   */
+  static manuallyMarkTaxFormAsReceived = (
+    payee: Collective,
+    user: User,
+    documentLink: string,
+    {
+      UserTokenId,
+      year,
+    }: {
+      UserTokenId?: number;
+      year?: number;
+    } = {},
+  ) => {
+    return sequelize.transaction(async sqlTransaction => {
+      let legalDocument = await LegalDocument.findOne({
+        where: { CollectiveId: payee.id, requestStatus: LEGAL_DOCUMENT_REQUEST_STATUS.REQUESTED },
+        lock: true,
+        transaction: sqlTransaction,
+      });
+
+      const attributes = {
+        service: LEGAL_DOCUMENT_SERVICE.OPENCOLLECTIVE,
+        requestStatus: LEGAL_DOCUMENT_REQUEST_STATUS.RECEIVED,
+        documentLink,
+        year: year || new Date().getFullYear(),
+      };
+
+      if (legalDocument) {
+        await legalDocument.update(attributes, { transaction: sqlTransaction });
+      } else {
+        legalDocument = await LegalDocument.create(
+          { ...attributes, CollectiveId: payee.id },
+          { transaction: sqlTransaction },
+        );
+      }
+
+      // This won't trigger any email
+      await Activity.create({
+        type: activities.TAXFORM_RECEIVED,
+        UserId: user.id,
+        CollectiveId: payee.id,
+        FromCollectiveId: user.CollectiveId,
+        UserTokenId: UserTokenId,
+        data: {
+          service: legalDocument.service,
+          document: legalDocument.info,
+          account: payee.info,
+          isManual: true,
```

Comment:
I think you can skip that commend with a `notify: false` flag.
```suggestion
          isManual: true,
          notify: false,
```

Reply:
This activity is never sent directly in `server/lib/notifications/email.ts`, only manually. The `isManual` flag is only there for us to easily differentiate when a tax form is submitted manually (for analytics)